Kernel/System/Console/Command/Maint/Elasticsearch/Migration.pm

```diff
@@ -734,7 +744,12 @@ sub _ExpandTemplate {
         return;
     }
     elsif ( IsNumber($Node) ) {
-        return $Node;
+        if ( defined( $Config->{ $Param{Key} } ) ) {
+            return $Config->{ $Param{Key} };
+        }
+        else {
+            return $Node;
+        }
```

Comment:
This whole block is likely not right, just because it is connected to my former comment.

Reply:
Did you test this and experienced a wrong behavior - which I would like to see -  or do you just guess?